source/features/rgh-improve-new-issue-form.tsx

```diff
@@ -18,6 +20,14 @@ function init(): void {
 			Clear cache
 		</button>,
 	);
+	void expectTokenScope('repo').catch(() => {
+		select('#issue_body_template_name')!.before(
+			<div className="flash flash-warn m-2">
+				Your Personal Access Token is either missing, incorrect or expired. Some features will not work without it.<br/>
+				You can update it <a href="#" onClick={openOptions as unknown as React.MouseEventHandler}>in the options</a>.
```

Comment:
Maybe "you can update it in the extension options"

Not sure if it's needed.

Reply:
I added a mention to RGH to clarify what the notice is about.